src/data/markdown/translated-guides/en/02 Using k6/02 Metrics/100 Create custom metrics.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Create custom metrics
+description: How to build custom k6 metrics for each metric type.
+---
+
+Besides the [built-in metrics](/results-output/metrics-reference), you can create custom metrics.
+For example, you can create some derived metrics from the built-in metrics, or use the [Response.timings](/javascript-api/k6-http/response) object to create a metric for a specific set of endpoints.
+
+Each metric type has a [constructor](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/constructor) to create a custom metric.
+The constructor creates a metric object of the declared type. Each type has methods to take metric measurements.
+
+If you need help constructing a custom metric, read the following sections of this document.
+They document the procedure and provide examples.
```

Comment:
Why are we adding examples for each type in this page? Isn't it better to have them in the dedicated doc for each type?

Reply:
We do also have examples in the metrics reference docs. It is indeed redundant, but I'm trying to think of how someone who's looking to make a custom metric may arrive in one of two places (here or the reference).

It's also at the end, so the most important information is at the top.

Not sure though, maybe it'd be better to just delete and remove the duplication... :thinking:

Reply:
Maybe one could be a link to the other?

Reply:
I think it's better, more efficient.

Here's a nice red diff: https://github.com/grafana/k6-docs/pull/1151/commits/7de40f7550cdf61df619ecd76a4bf3b7b4617098